account_vat_period_end_statement/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,45 @@
+**Italiano**
+
+    Per fare la liquidazione IVA, aprire Fatturazione > Contabilità > Liquidazioni IVA, il menù è visibile solo quando è abilitato il gruppo 'Mostrare funzionalità contabili complete'.
+    Selezionare un registro che conterrà le registrazioni contabili della liquidazione.
+    Il campo 'Conto IVA Erario' contiene il conto dove verrà effettuata la registrazione della liquidazione IVA.
+
+    L'oggetto 'Liquidazione IVA' permette di specificare ogni importo e il conto utilizzato dalla liquidazione.
+    Di norma, gli importi di debito e credito delle tasse vengono caricati automaticamente dai periodi selezionati
+    (vedere Configurazione per generare correttamente i periodi).
+    I debiti e crediti precedenti vengono caricati dalle liquidazioni IVA precedenti, in base allo stato del loro pagamento.
+
+    Per creare la registrazione contabile, cliccare sul bottone 'Crea movimento', dentro il tab 'Conti'.
```

Comment:
s/cliccare/fare clic
s/bottone/pulsante
s/tab/scheda

Reply:
Se non è un problema, preferisco mantenere 'cliccare', per il resto ok.

Reply:
Come detto in https://github.com/OCA/l10n-italy/pull/1176#discussion_r277571193, cambiato in _fare clic_